plugins/bgp/model/keys.go

```diff
@@ -0,0 +1,25 @@
+package model
+
+import "github.com/ligato/vpp-agent/pkg/models"
+
+const ModuleName = "bgp"
+
+var (
+	ModelBgpGlobal = models.Register(&GlobalConf{}, models.Spec{
+		Module:  ModuleName,
+		Version: "v1",
+		Type:    "global",
+	})
```

Comment:
Since there is no `Name` attribute for `GlobalConf` (as it is just one instance), the resulting key for this definition would lack suffix - i.e. it would end with "/" like a directory path. What we do to avoid this, is to add a static key suffix "settings":
https://github.com/ligato/vpp-agent/blob/master/api/models/vpp/nat/keys.go#L31
The resulting key will be then: `config/bgp/v2/global/settings`